### DISCUSSION TITLE
Validation message as Toast

### DISCUSSION THREAD

**Discussion by klaarseSICKAG** [synergy-design-system/synergy-design-system#647] [RFCs]
Instead of an inline alert, another option could be showing the alert as a toast.
Advantage: don't use the content space.

**Comment by schilchSICKAG** [comment]
I think we could do this, but there are a couple of issues with this:

1. The developer would need to set up the toast stack (we could create a demo for this or even provide it via a prop?)
2. You will have to create a link between the toast and the invalid input
3. How do we control the toast stack on revalidation? We would have to highlight the error again, but maybe not add another new one to the stack, as we do not know which of the currently shown toasts is coming from validation.

Lets prototype this in the future
